docs/app/index.ejs

```diff
@@ -67,6 +67,7 @@
   <script src="//cdnjs.cloudflare.com/ajax/libs/react/<%= htmlWebpackPlugin.options.versions.react %>/react<%= __PROD__ ? '.min' : '' %>.js"></script>
   <script src="//cdnjs.cloudflare.com/ajax/libs/react/<%= htmlWebpackPlugin.options.versions.reactDOM %>/react-dom.min.js"></script>
   <script src="//cdnjs.cloudflare.com/ajax/libs/react/<%= htmlWebpackPlugin.options.versions.reactDOM %>/react-dom-server.min.js"></script>
+  <script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
```

Comment:
This script tag is in the body so there is no need to make it async since it cannot block body content from loading.  Let's remove the `async` attribute.

Regarding [this comment](https://github.com/Semantic-Org/Semantic-UI-React/pull/1292/files#r100858877), the `adsbygoogle` object's custom push method should be called only once in another script tag immediately following this one.  I've used a recursive timeout to wait until the library has loaded:

```html
<script>
  var pushGoogleAd = (function() {
    setTimeout(function () {
      if (window.adsbygoogle && window.adsbygoogle.loaded) window.adsbygoogle.push({})
      else pushGoogleAd()
    }, 1000)
  }())
</script>
```